deploy/common/local-dev-values.yaml

```diff
@@ -1,44 +1,73 @@
 # Reduced resource overrides for the stackrox-central-services Helm chart that allow deploying
 # on Docker for Desktop on a typical developer box.
-# Please keep /operator/tests/*/*/*central-cr.yaml in sync
```

Comment:
Why remove this pointer? Shouldn't we now have pointers to the remaining 3 places in all 4 places now instead?

Reply:
Added the comment in all places. Didn't add to secured cluster since there is no local-dev-values.yaml and the kuttle resources are currently very small, so I'm not sure if they are supposed to be the same.